WebService/ComparisonEngineUI/ComparisonEngineUI/Constants.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ComparisonEngineUI
+{
+    class Constants
+    {
+        public static string BarsUrl = "https://localhost:5001/api/Bars";
+        public static string DrinksUrl = "https://localhost:5001/api/Drinks";
+        public static string SpecificPricesUrl = "https://localhost:5001/api/SpecificPrices";
+    }
+}
```

Comment:
could we move this to appsettings.json?

Reply:
I don't know what would be the equivalent of appsettings in xamarin.